charts/images.yaml

```diff
@@ -37,20 +37,57 @@ images:
   repository: registry.k8s.io/ingress-nginx/controller
   tag: "v0.49.3"
   targetVersion: "< 1.22"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'public'
+      authentication_enforced: true
```

Comment:
This is the image of the nginx addon, isn't it?
If yes, we don't enforce authentication to my knowledge. Customers are free to expose whatever they want with whatever auth they see fit (even no auth). Hence `false` is likely more appropriate here.

Reply:
From what I see in the file we have another entry for the nginx addon where two different versions are used - [ref](https://github.com/gardener/gardener/blob/master/charts/images.yaml#L289-L300)

Reply:
Ah, it's the seed one, sorry, then this here is fine.